ravindu14/SliitSpmProject
# Feature/task id #1025 basic interface

how to test
1. run "npm install".
2. run the application using "ng serve" command
3. navigate to "http://localhost:4200/tables" url in browser
4. click on "save" button without filling name,nic and upload form fields. (an error message should be displayed)
5. click on reset button to reset
6. fill name,nic current year, semester and upload fields and click on save
